Traceback (most recent call last):
  File "/home/gilberto/Desktop/CoreTemp/tray.py", line 92, in <module>
    win.mainloop()
  File "/usr/lib/python3.10/tkinter/__init__.py", line 1458, in mainloop
    self.tk.mainloop(n)
KeyboardInterrupt
Exception in Tkinter callback
Traceback (most recent call last):
  File "/usr/lib/python3.10/tkinter/__init__.py", line 1921, in __call__
    return self.func(*args)
  File "/usr/lib/python3.10/tkinter/__init__.py", line 839, in callit
    func(*args)
TypeError: 'str' object is not callable
